pkg/execution/executor/executor.go

```
@@ -2407,6 +2407,10 @@ func (e *executor) AppendAndScheduleBatch(ctx context.Context, fn inngest.Functi
 				"account_id": bi.AccountID.String(),
 			},
 		})
+
+		if err := e.batcher.TrackBatchCreate(ctx, bi.AccountID); err != nil {
+			return fmt.Errorf("could not track batch create: %w", err)
```

minor: should we return this error or just log it?